tests/integration_tests/performance/test_benchmarks.py

```diff
@@ -17,7 +17,7 @@
 
 
 @pytest.mark.no_block_pr
-@pytest.mark.timeout(600)
+@pytest.mark.timeout(900)
```

Comment:
Mhhh, I think it's possible. We'd need to have the fixture run `cargo bench --all -- --list`, but then we'd also need to rework the test itself quite a bit I think. I'll open an issue to do this maybe, so that we can merge this to fix the spurious failures, if that's okay? :o